### DISCUSSION TITLE
how to enable/disable landclaims

### DISCUSSION THREAD

**Discussion by Tomislav-2000** [GriefPrevention/GriefPrevention#1886] [Q&A]
when i downloaded the plugin it worked however, on my server i have multyverse and when i joined the survival world it said that land claims are not enabled on this world, however i cant seem to find how to enable it on my world. please tell me the command

**Comment by Jikoo** [comment]
https://docs.griefprevention.com/configuration/#restricting-where-claims-may-be-created

You need to configure the plugin, it's not a command.
